### ISSUE TITLE
Package.json

### ISSUE THREAD

**Issue by Chinedum1** [insidesherpa/JPMC-tech-task-2-PY3#423]
![Screen Shot 2020-07-12 at 12 21 44 AM](https://user-images.githubusercontent.com/43798273/87238819-aca27600-c3d5-11ea-90ca-9bdefb94d259.png)

Does any know what my error could possibly be? I'm working on a Mac and I've tried looking at the slides, but I can't figure out what they mean by there not being a package.json file when I run npm install or npm start. Shouldn't the JSON file already be downloaded when we started the project in the beginning?

**Comment by shubhankar-naik** [comment]
I think you weren't inside the folder when you wrote npm install. First go inside the folder using cd foldername and then npm install there.

**Comment by Chinedum1** [comment]
@shubhankar-naik Are you referring to the JPMC-tech-task-2-py3 folder we cloned at the beginning?

**Comment by shubhankar-naik** [comment]
@Chinedum1  Yes that's the folder write "cd JPMC-tech-task-2-py3"  in the terminal then write npm install
